praw/reddit.py

```diff
@@ -83,6 +84,23 @@ def read_only(self, value: bool) -> None:
         else:
             self._core = self._authorized_core
 
+    @property
+    def validate_on_submit(self) -> bool:
+        """Get validate_on_submit."""
+        val = self._validate_on_submit
```

Comment:
Can we avoid using shortened variable names please? Consider `value` instead.

Reply:
Ok.